signature/src/signatures/sigstore.rs

```diff
@@ -11,9 +11,7 @@ use std::collections::HashMap;
 use std::ffi::OsStr;
 use std::fs;
 
-// FIXME: Now, for Demo, the dir which the sigstore config files locate
-// is hardcoded.
-pub const SIGSTORE_CONFIG_DIR: &str = "/etc/containers/registries.d";
+pub const SIGSTORE_CONFIG_DIR: &str = "/run/image-security/simple_signing/sigstore_config";
```

Comment:
This may be beyond the scope of this PR. at present, the location of the configuration file of this PR is consistent with the ontology of image-rs. Maybe we can discuss and formally modify this configuration file directory in a new issue and PR？